shared/src/business/useCases/processStreamRecordsInteractor.js

```diff
@@ -294,10 +363,9 @@ const processRemoveEntries = async ({ applicationContext, removeRecords }) => {
   });
 
   if (failedRecords.length > 0) {
-    applicationContext.logger.error(
-      'the records that failed to delete',
+    applicationContext.logger.error('the records that failed to delete', {
```

Comment:
We seem to be re-using this throughout this file. Adding something to Trello backlog to address.